WooCommerce/UITestsFoundation/Screens/Login/TwoFAScreen.swift

```diff
@@ -0,0 +1,38 @@
+import ScreenObject
+import XCTest
+
+public final class TwoFAScreen: ScreenObject {
+
+    private let two2FAFieldGetter: (XCUIApplication) -> XCUIElement = {
```

Comment:
looks like a typo with both `two` and `2` in the name

```suggestion
    private let twoFAFieldGetter: (XCUIApplication) -> XCUIElement = {
```